geometry/Building.cpp

```diff
@@ -162,1287 +161,1262 @@ void Building::SetCaption(const std::string& s)
-                        // remove
-                        // duplicates fromWllPiecs
-                        if(!WallPieces.empty())
-                        {
-                              //remove  duplicaes from wallPiecess
-
-                              auto end = WallPieces.end();
-                              for (auto it = WallPieces.begin(); it != end; ++it) {
-                                    end = std::remove(it + 1, end, *it);
-                              }
-                              WallPieces.erase(end, WallPieces.end());
+                    // getc(stdin);
+                } // while
+                // remove
+                // duplicates fromWllPiecs
+                if(!WallPieces.empty()) {
+                    // remove  duplicaes from wallPiecess
+
+                    auto end = WallPieces.end();
+                    for(auto it = WallPieces.begin(); it != end; ++it) {
+                        end = std::remove(it + 1, end, *it);
+                    }
+                    WallPieces.erase(end, WallPieces.end());
 #if DEBUG
-                         std::cout << "..removing duplicates pieces..\n";
-                         for(auto t: WallPieces){
-                              std::cout << ">>>> Piece: " << std::endl;
-                              t.WriteToErrorLog();
-                         }
-#endif
-                         // remove big wall and add one wallpiece to walls
-                         ReplaceBigWall(subroom.second, bigWall, WallPieces);
+                    std::cout << "..removing duplicates pieces..\n";
+                    for(auto t : WallPieces) {
+                        std::cout << ">>>> Piece: " << std::endl;
+                        t.WriteToErrorLog();
                     }
-               }// bigLine
-          }//s
-     }//r
-
-     if(removed)
-     {
-          auto geometryFile = add_prefix_to_filename(
-            "correct_", GetGeometryFilename());
+#endif
+                    // remove big wall and add one wallpiece to walls
+                    ReplaceBigWall(subroom.second, bigWall, WallPieces);
+                }
+            } // bigLine
+        }     // s
+    }         // r
 
-          if(SaveGeometry(geometryFile)) {
-              GetConfig()->SetGeometryFile(geometryFile);
-          }
+    if(removed) {
+        auto geometryFile = add_prefix_to_filename("correct_", GetGeometryFilename());
 
-     }
+        if(SaveGeometry(geometryFile)) {
+            GetConfig()->SetGeometryFile(geometryFile);
+        }
+    }
 
-    auto t_end = std::chrono::high_resolution_clock::now();
-    double elapsedTimeMs = std::chrono::duration<double, std::milli>(t_end-t_start).count();
+    auto   t_end         = std::chrono::high_resolution_clock::now();
+    double elapsedTimeMs = std::chrono::duration<double, std::milli>(t_end - t_start).count();
     Log->Write("INFO:\tLeave geometry correct with success (%.3f s)", elapsedTimeMs);
     return true;
 }
```

Comment:
I would suggest closing this PR first since it's BIG. 
Then, fix the comments peu a peu.

Reply:
I would suggest closing this PR first since it's BIG.
Then, fix the comments peu a peu.

> First step merging this or finishing #505?

I would suggest closing this PR first since it's BIG.
Then, fix the comments peu a peu.

Reply:
No, clang format can only restrict the amount of empty lines, but not add new ones.